claat/render/md.go

```diff
@@ -148,9 +148,18 @@ func (mw *mdWriter) text(n *types.TextNode) {
 func (mw *mdWriter) image(n *types.ImageNode) {
 	mw.space()
 	mw.writeString("![")
-	mw.writeString(path.Base(n.Src))
+	if n.Alt != "" {
+		mw.writeString(n.Alt)
+	} else {
+		mw.writeString(path.Base(n.Src))
+	}
 	mw.writeString("](")
 	mw.writeString(n.Src)
+	if n.Title != "" {
+		mw.writeString("\"")
+		mw.writeString(n.Title)
+		mw.writeString("\"")
```

Comment:
I think you're missing a space before `"title text"`. Also, probably safer to use quoted string:

```go
if n.Title != "" {
  mw.writeString(fmt.Sprintf(" %q", n.Title))
}
```

Reply:
fixed